From 01d02a4468bc895f088bc143c7f957642f1d8f8e Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 28 Jun 2018 10:32:44 +0800
Subject: [PATCH] ARM: rk3308_linux_aarch32[_debug]_defconfig: enable
 KERNEL_LZ4

Compress kernel use LZ4 for faster boot time.
zImage larger 15~20%, but save about 75% time. For example:
             size(byte) time(s)
zImage (gz)  2891776    0.177
zImage (lz4) 3334144    0.042

Change-Id: I6a65569bbfc41eb63ee4154df26d055af7aa4ce7
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rk3308_linux_aarch32_debug_defconfig | 1 +
 arch/arm/configs/rk3308_linux_aarch32_defconfig       | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig b/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig
index c72571d51e4e..099793896ea6 100644
--- a/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig
+++ b/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig
@@ -1,4 +1,5 @@
 # CONFIG_LOCALVERSION_AUTO is not set
+CONFIG_KERNEL_LZ4=y
 CONFIG_DEFAULT_HOSTNAME="localhost"
 # CONFIG_SWAP is not set
 CONFIG_SYSVIPC=y
diff --git a/arch/arm/configs/rk3308_linux_aarch32_defconfig b/arch/arm/configs/rk3308_linux_aarch32_defconfig
index 8bd80726f41b..cd0f49323385 100644
--- a/arch/arm/configs/rk3308_linux_aarch32_defconfig
+++ b/arch/arm/configs/rk3308_linux_aarch32_defconfig
@@ -1,4 +1,5 @@
 # CONFIG_LOCALVERSION_AUTO is not set
+CONFIG_KERNEL_LZ4=y
 CONFIG_DEFAULT_HOSTNAME="localhost"
 # CONFIG_SWAP is not set
 CONFIG_SYSVIPC=y
-- 
2.35.3

